### ISSUE TITLE
Name issue on Ideascube catalog

### ISSUE THREAD

**Issue by fheslouin** [ideascube/ideascube#361]
Downloading and installing mullah-piaz-digest.fa lead to a name issue (http://catalog.ideascube.org/kiwix.yml.html)
```

(1:298)$ ideascube catalog install mullah-piaz-digest.fa                                                                                                                                          
IDEASCUBE_ID=idb_fra_grandesynthe
Importing settings from ideascube.conf.idb_fra_grandesynthe

Installing mullah-piaz-digest.fa
Rebuilding the Kiwix library
Restarting service kiwix-server
```
```
(1:299)$ ll /var/cache/ideascube/catalog/packages/
total 239040
-rw-r--r-- 1 ideascube ideascube 244776958 juin  13 18:53 mullah-piaz-digest.fa-2012-12-18
```
Renaming does not work as it should
```
(1:300)$ ll /var/ideascube/kiwix/data/content/
-rw-r--r-- 1 ideascube ideascube    50470643 juin  13 18:58 mullah_piaz_digest.zim
```

```
(1:301)$ ll /var/ideascube/kiwix/data/index/
drwxr-xr-x 2 ideascube ideascube 4096 juin   8 22:01 mullah_piaz_digest.zim.idx
```

```
<book id="d3756db6-f994-6714-90b2-d4c381e9efba" path="data/content/mullah_piaz_digest.zim" indexPath="data/index/mullah_piaz_digest.zim.idx" indexType="xapian" title="Mullah Piaz" description="Digest" language="per,eng" creator="Mansour Baradari" publisher="Mansour Baradari" favicon=
```
Same issue with `icd10.fr`

**Comment by yohanboniface** [comment]
I'm not seeing the issue, but it's certainly me not up to date: what were you expecting instead?

**Comment by yohanboniface** [comment]
I think this has been fixed by #450.
